.travis.yml

```diff
@@ -0,0 +1,40 @@
+sudo: false
+language: python
+python:
+  - '2.7'
+  - '3.4'
+  - '3.6'
+matrix:
+    include:
+      # This is the ltd-mason documentation deployment build
+      - python: "3.5"
+        env: LTD_MASON_BUILD=true
+install:
+  - pip install -r requirements.txt
+script:
+  - py.test --flake8 --cov=ltdconveyor
```

Comment:
Just as an FYI - secure vars aren't available if the PR is from a fork. Travis sets the env var `TRAVIS_SECURE_ENV_VARS` that you can test for.